commands/bank/balance.js

```diff
@@ -0,0 +1,35 @@
+const logger = require('../../utils/logger');
+const getDateTimeStringLocal = require('../../utils/getDateTimeStringLocal');
+const fs = require('fs');
+
+module.exports = {
+    data: {
+        name: 'balance',
+        type: 'text'
+    },
+    async execute(message) {
+        const guildId = message.guildId;
+        const bankData = require('../../data/bank.json');
+        
+        logger.log(`!claim user: ${message.member.user.username} | guildId: ${guildId}`);
+        
+        let outputStr = '';
+        let userId = message.member.user.id;
+        let userName = message.member.user.username;
+        let timestampString = getDateTimeStringLocal();
+        if (!bankData.user_ledger[userId]) {
+            bankData.audit_log[timestampString] = `${userName} has requested their balance, but has no Cullivery account.`;
```

Comment:
camel case should be used exclusively since all code is javascript (`user_ledger` -> `userLedger`, etc)